djconnectwise/sync.py

```diff
@@ -3157,6 +3188,37 @@ def filter_by_record_type(self):
         return self.model_class.objects.filter(
             record_type__in=project_record_types)
 
+    def _instance_ids(self, filter_params=None):
+        tickets_qset = self.filter_by_record_type()
+
+        settings = DjconnectwiseSettings().get_settings()
+        keep_closed = settings.get('keep_closed_ticket_days')
```

Comment:
You don't need to check keep_closed_ticket_days for project tickets, thats already handled by the project models. This change should just check if a project exists for a project ticket, and if it doesnt then we skip the ticket. The logic for keeping the ticket can completey cascade to the project logic.